math/prime_numbers.cpp

```diff
@@ -3,23 +3,23 @@
 
 std::vector<int> primes(int max) {
     max++;
-    std::vector<int> res;
+    std::vector<int> vec;
     std::vector<bool> numbers(max, false);
-    for (int i = 2; i < max; i++) {
+    for (int i = 2; i < max; i++) {   //To check till the given input
         if (!numbers[i]) {
-            for (int j = i; j < max; j += i)
+            for (int j = i; j < max; j = j+i)
                 numbers[j] = true;
-            res.push_back(i);
+            vec.push_back(i); //To push the value to vector
```

Comment:
This comment adds no value because the code already says the same thing.

The same is true of all the comment below.  They add no new information that is not already evident from reading the code.